### ISSUE TITLE
Rescaling satbias_pc files

### ISSUE THREAD

**Issue by gmao-msienkie** [GEOS-ESM/GEOSana_GridComp#20]
When the characteristics of radiance observations from a satellite instrument change abruptly, increasing the background error for the bias correction coefficients allows them to more rapidly adjust to more appropriate values.  The values read from the satbias_pc file serve both as a matrix preconditioner and as the background error for the bias coefficients.  
The make_pcscaling.x program reads the satbias_pc file and if the date matches an entry in the namelist, it multiplies the specified satellite/instrument/channel by a specified factor.

There are two issues here - one immediate and one ongoing.

The immediate issue is that there is code that needs to be added to the GSI 'analyzer' to run the make_pcscaling.x routine so that the updates to the satbias_pc can be automatically taken care of while the DAS executes.

The ongoing issue is that the namelist file should be updated with known cases where the pcscaling needs to be applied on a particular date to the values for a particular instrument.

**Comment by gmao-msienkie** [comment]
I updated the pcscaling.nml file with some real cases - two cases I had found on the satellite monitoring page and also the configuration that Mohar was testing.  
I removed the line that Jing was using for testing.  That means the testing target in his GNUmakefile in CVS wouldn't work as he intended.  However his test code also had not been copied to the Git build either.   Presumably it could be brought over and his test case added to the namelist in the build but I worry that it could be confusing for others to see it in the master namelist file.
Of course per our discussion Tuesday Mohar is going to run a test where she brings in the different satellite data on different days so if that is implemented we would need to modify the namelist accordingly.
